src/org/opendatakit/briefcase/export/ExportAction.java

```diff
@@ -72,13 +72,14 @@ public static void export(BriefcaseFormDefinition formDefinition, ExportConfigur
           .orElseThrow(() -> new RuntimeException("PEM file not present"))
       ).get());
     }
+    boolean overwriteExistingFiles = configuration.getOverwriteExistingFiles().isPresent() ? configuration.getOverwriteExistingFiles().get() : false;
     ExportToCsv action = new ExportToCsv(
         terminationFuture,
         configuration.getExportDir().orElseThrow(() -> new RuntimeException("Export dir not present")).toFile(),
         formDefinition,
         formDefinition.getFormName(),
         true,
-        false,
+        overwriteExistingFiles,
```

Comment:
This line should be `configuration.getOverwriteExistingFiles().orElse(false),`

Remember that if you are handling an `Optional<T>` and using `if`, it's very probable that you're not using it right.